addons/hr_expense/models/hr_expense.py

```diff
@@ -104,25 +109,19 @@ def _get_employee_id_domain(self):
 
     is_editable = fields.Boolean("Is Editable By Current User", compute='_compute_is_editable')
     is_ref_editable = fields.Boolean("Reference Is Editable By Current User", compute='_compute_is_ref_editable')
-    product_has_cost =  fields.Boolean("Is product with non zero cost selected", compute='_compute_product_has_cost')
+    product_has_cost = fields.Boolean("Is product with non zero cost selected", compute='_compute_product_has_cost')
+    product_has_tax = fields.Boolean("Whether tax is defined on a selected product", compute='_compute_product_has_cost')
     same_currency = fields.Boolean("Is currency_id different from the company_currency_id", compute='_compute_same_currency')
     duplicate_expense_ids = fields.Many2many('hr.expense', compute='_compute_duplicate_expense_ids')
 
     sample = fields.Boolean()
-    label_total_amount_company = fields.Char(compute='_compute_label_total_amount_company')
     label_convert_rate = fields.Char(compute='_compute_label_convert_rate')
 
-    @api.depends("product_has_cost")
+    @api.depends('product_has_cost')
```

Comment:
Not sure this is necessary. Some have double, some have single. Changing only one of them is just making the diff bigger.

Reply:
I don't do this often.. it's just from my first task ever and it annoyed me that it was inconsistent with the rest of the file; That's why :innocent: